### PR TITLE
Improve usability at new organization creation

### PR DESCRIPTION
**Pull request by micael-grilo**
This is a PR with a improvement to the search organization button on administration. The search results appear as you type and clicking on the buttons reloads the page unnecessarily causing some confusion to the users, so I've removed this page reloading feature.

It adds also a 'Find or create your organization' button so the user can click on it when no organization results have been found. 

Before:
image

After:
image

Best!

### CODE REVIEW

**Comment by taniki** [comment]
Thanks for the PR! This sounds great. You can add a screenshot (before/after) for cosmetical improvements. It is easier for quick reviews.

**Comment by micael-grilo** [comment]
Updated with screenshots! :)

File: js/components/organization/card-filter.vue [hunk 1]
```diff
@@ -13,9 +13,9 @@
                 <input type="text" name="search" class="form-control"
                     :placeholder="_('Search')" v-model="search_query" />
                 <div class="input-group-btn">
-                    <button type="submit" name="submit" class="btn btn-warning btn-flat">
+                    <div class="btn btn-warning btn-flat">
```
Comment (abulte) [inline js/components/organization/card-filter.vue]: Why did you remove the `button`?

Comment (micael-grilo) [reply js/components/organization/card-filter.vue]: The button is just reloading the page and you lose all your search criteria when you click on it, it does nothing as the search happens when you are typing so no need for a submit button. ( Users get confused when they click on it, the page reloads and the search gets lost.)
